tests/Graph/Nodes/SplitTest.php

```diff
@@ -114,4 +113,21 @@ public function n() : void
     {
         $this->assertSame(self::N, $this->node->n());
     }
+
+    /**
+     * @test
+     */
+    public function cleanup() : void
```

Comment:
Here you seem to be testing the cleanup behaviour of the nodes. But initially it reads as a cleanup for the test case. So "testCleanup" might be less confusing. Second, the functionality you're testing makes sense to test but why is it in the tests for Split rather than tests for nodes?

Reply:
Yeah we use PHPUnit annotations rather than prefixing the test method with the word `test` so for consistency I'd rather stick to the plan, although, I agree one who may be unfamiliar with PHPUnit might mistake this for some sort of method that resets or cleans up stuff left over from a test.

As for why the unit test lives in the Split class test, no reason other than we needed somewhere to put it. Perhaps I'll add this test to the rest of the nodes that implement this method as well.